### ISSUE TITLE
How to pass job variables to job.play()

### ISSUE THREAD

**Issue by Prophet32j** [jdalrymple/gitbeaker#2731]
**Description**
There doesn't appear to be an ability to pass variables to a pipeline when playing from the gitbeaker API. The Gitlab Jobs API shows it as the payload in the call: https://gitlab.com/gitlab-org/gitlab/-/blob/master/doc/api/jobs.md#run-a-job

I need to pass in a couple variables which will simplify my entire process.

Something like this is what I would expect with this API:

```javascript
const api = new Gitlab({
  host: myHost,
  token: myToken
})

export async function playJob(job: Types.JobSchema, projectId: String): Promise<void> {
  await api.Jobs.play(projectId, job.id, {
    jobVariableAttributes: [
      {
        key: 'CUSTOM_JOB_VARIABLE_NAME',
        value: 'CUSTOM_JOB_VARIABLE_VALUE'
      }
    ]
  })
}
```

**Checklist**
- [x] I have checked that this is not a duplicate issue.
- [x] I have read the documentation.

**Comment by jdalrymple** [comment]
Hmm, that should work. I've had trouble in the past with the way the Gitlab API handles array though, since they didnt seem to be consistent. Ill give it a look and follow up when I have the time.

**Comment by Prophet32j** [comment]
If I get time this weekend I’ll look into contributing with a PR. This is a
pretty solid API wrapper. Making it align to the Gitlab API more closely
will only make it better.

Thanks.

**Comment by jdalrymple** [comment]
Fixed for the next release!

**Comment by Prophet32j** [comment]
Wow! That is awesome. I failed to find time. I’ll upgrade when it’s
available and test it out!

**Comment by Prophet32j** [comment]
@jdalrymple I see this is tagged for the next major release. When are you intending to release?

**Comment by jdalrymple** [comment]
End of the month! :crossed_fingers: At least the rc will be available then for people to test out. Currently just need to fix the CI pipeline for the automated release and testing.

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`37.0.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/37.0.0) :rocket:
